conf/machine/include/tegra194.inc

```diff
@@ -39,3 +39,5 @@ L4T_DEB_SOCNAME = "t194"
 NVPMODEL ?= "nvpmodel_t194"
 NVPOWER ?= "jetsonpower_t194"
 NVFANCONTROL ?= "nvfancontrol_p2888"
+
+MACHINE_EXTRA_RRECOMMENDS += "kernel-module-tegra-udrm"
```

Comment:
should tegra234 get a `kernel-module-nvidia-drm` like this, or is that what is not possible because of the `KERNEL_SPLIT_MODULES=0` on the `nvidia-display-driver`?

Reply:
Yes, That's true
For now, we have `MACHINE_EXTRA_RRECOMMENDS += "nvidia-display-driver"` in the `tegra194.inc`
I will rework it at some point to have the kernel modules separate something like: 
```
kernel-module-nvidia
kernel-module-nvidia-drm
kernel-module-nvidia-modeset
```